### PR TITLE
Port decorative block to 1.11

### PR DESCRIPTION
**Pull request by MineShadow7**
Port 1.10 block and tools recipes to 1.11.

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1377/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 36 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 33 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecipeHandler.java#L99](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L99): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L105](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L105): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L124](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L124): Remove this unused private "addHutRecipe" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requestsystem](https://github.com/Minecolonies/minecolonies/blob/a4291ab7b9805c719bfcc0b64c22d43dcba98c73/src/main/java/com/minecolonies/coremod/colony/requestsystem): Add a 'package-info.java' file to document the 'requestsystem' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by marchermans — APPROVED** [review]

**Comment by Asherslab** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1377/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

File: src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java [hunk 1]
```diff
@@ -30,8 +30,8 @@
     public static final Block blockHutWareHouse           = new BlockHutWareHouse();
     public static final Block blockConstructionTape       = new BlockConstructionTape();
     public static final Block blockConstructionTapeCorner = new BlockConstructionTapeCorner();
-    public static final Block blockRack                    = new BlockMinecoloniesRack();
-
+    public static final Block blockTimberFrame            = new BlockTimberFrame();
+    public static final Block blockRack                   = new BlockMinecoloniesRack();
 
     // Deactivated for now
     // public static final Block blockBarrel        = new BlockBarrel();
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java:37]: MINOR This block of commented-out lines of code should be removed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 36 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 33 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecipeHandler.java#L98](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L98): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L104](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L104): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L123](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L123): Remove this unused private "addHutRecipe" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [requestsystem](https://github.com/Minecolonies/minecolonies/blob/11ab09c17700104592d295b311b94aa081940828/src/main/java/com/minecolonies/coremod/colony/requestsystem): Add a 'package-info.java' file to document the 'requestsystem' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**